resources/views/layouts/navbar.blade.php

```diff
@@ -1,5 +1,7 @@
 @vite(['resources/sass/navbar.scss'])
 @vite(['resources/js/logout_modal.js'])
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"
```

Comment:
We don't need a CDN of bootstrap since we already have installed bootstrap within the Laravel

Reply:
It was there when I opened the nabvar's blade file. I don't know who has written it! How do I do it?

Reply:
I discussed it with my batch member. 
He uses those when creating the logout feature.
But if it is on the navbar's blade, the user cannot use the navbar's feature.
He agrees it's deleted on there. Because it's still running when it was commented out.